packages/js/components/src/experimental-tree-control/tree.tsx

```diff
@@ -50,13 +46,11 @@ export const Tree = forwardRef( function ForwardedTree(
 						index={ index }
 						// Button ref is not working, so need to use CSS directly
 						onLastItemLoop={ () => {
-							if ( isCreateButtonVisible ) {
-								(
-									thisRef.current?.parentElement?.querySelector(
-										'.experimental-woocommerce-tree__button'
-									) as HTMLButtonElement
-								 )?.focus();
-							}
+							(
+								document.querySelector(
```

Comment:
I'm having a hard time improving this, because the tree is recursive (TreeItems can have Trees), so I can't know the exact DOM structure. It would help a lot if I could get the ref to the button, but that's not working. Any ideas?